lxd/resource_lxd_publish_image.go

```diff
@@ -24,6 +24,21 @@ func resourceLxdPublishImage() *schema.Resource {
 				ForceNew: true,
 				Required: true,
 			},
+			"fingerprint": {
+				Type:     schema.TypeString,
+				ForceNew: false,
+				Computed: true,
+			},
+			"architecture": {
+				Type:     schema.TypeString,
+				ForceNew: false,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeInt,
+				ForceNew: false,
+				Computed: true,
+			},
```

Comment:
Thank you for adding these. Please make sure they're included in the documentation, too.

Reply:
Documentation updated.